deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/kyvernoMountLocalAwsCreds.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if and .Values.conditionalPackageInstalls.kyverno .Values.kyvernoPolicies.mountLocalAwsCreds }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-kyverno-aws-creds-policy
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "9"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+data:
+  policy.yaml: |
+    apiVersion: kyverno.io/v1
+    kind: ClusterPolicy
+    metadata:
+      name: mount-local-aws-creds
+      annotations:
+        policies.kyverno.io/title: Mount Local AWS Credentials
+        policies.kyverno.io/description: >-
+          Mounts local AWS credentials from host path into pods for local development.
+    spec:
+      rules:
+        - name: inject-aws-creds-volume
+          match:
+            any:
+              - resources:
+                  kinds:
+                    - Pod
```

Comment:
this is for ALL pods?  This means that if a developer gets tricked to run a nefarious pod in their k8s cluster, that nefarious pod could have access to much more than the user would have expected.

Can we scope this down to only pods with a particular annotation.  At least then, when a dev adds a resource, they're hopefully looking at the resource definition (which could always outright mount any files - so they had better be reading it!)